packages/terra-header/src/Header.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './Header.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Child element to be displayed on the right end of the header.
+   * The element passed as children will be decorated with flex attributes.
```

Comment:
Should probably clarify that this is expected to be and will be treated a collapsible menu view element.

Reply:
Addressed here https://github.com/cerner/terra-core/pull/2545/commits/95d3b46d474a4b5d9a278736bbd874b2d751115e.